src/webgl/texture.js

```diff
@@ -608,6 +626,21 @@ export default class Texture extends Resource {
   }
   */
 
+  // OVERRIDES
+  setParameters(parameters) {
```

Comment:
Yes they are in opts, not in opts.parameters. You need to extract them (there is a code example in the issue).